Content/posts/revealing-and-explaining-package-scores.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2023-10-31 12:00
+title: Revealing and explaining package scores
+description: When you search the Swift Package Index, the order in which the search results are displayed is determined by a combination of the relevance of text in the package name and description, and an internal score based on various metrics. Today, we're adding a feature that
+---
+
+This guest post is from [Cyndi Chin](https://cyndichin.github.io/) and announces the results of her work for this year’s [Swift Mentorship Program](https://www.swift.org/mentorship/).
+
+Working with Cyndi over the past twelve weeks has been an absolute pleasure. Her enhancements meaningfully push the Swift Package Index forward by making our internal package ranking algorithm transparent, allowing package authors to understand our search results better.
+
+One of Cyndi’s goals was to be involved in shipping something from beginning to end, and the end of that process is the announcement of the feature going live on the site! So, I’ll finish by saying a huge thank you to Cyndi for all of her hard work and hand you over to her for the rest of this post so she can make today’s announcement.
+
+---
+
+When you search the Swift Package Index, we determine the order in which we show search results by two primary factors: the relevance of the text in the package name and description and an internal score based on various metrics.
+
+Until now, this score was not publicly visible on the site, but as of today, we publish it publicly in a new [Package Score](https://swiftpackageindex.com/apple/swift-markdown/information-for-package-maintainers#package-score) section on every [Package Maintainer’s page](https://swiftpackageindex.com/apple/swift-markdown/information-for-package-maintainers). We also deployed two additional metrics that contribute to the score.
+
+### What is a package score?
+
+In combination with the relevancy of a search query, we use a package score to partially influence the ordering of search results on the Swift Package Index. The new information gives package authors and maintainers insight into how we calculate package scores, and the metrics that calculation relies on, such as how actively maintained a package is, whether it has documentation, tests, and various other factors.
+
+<picture class="shadow">
+  <source srcset="/images/package-maintainers-score-section~dark.png" media="(prefers-color-scheme: dark)">
+  <img src="/images/package-maintainers-score-section~light.png" alt="The package score section on the package maintainer's page showing a breakdown of package score for a package.">
+</picture>
+
+The most important thing about this package score is that we want it to
```

Comment:
do we want to continue this sentence, maybe?
`inform package maintainer's on their package score and provide visibility into how it is calculated.`

Reply:
Sounds great 👍